homework_unit_8/layout_bootstrap3/style.scss

```
@@ -0,0 +1,570 @@
+$bg-header: #4e5366;
+$text-white: #fff;
+$text-dark: #434b56;
+$text-gray: #939393;
+$color-blue: #4daaaf;
+$color-cyan: #6ac8ae;
+$color-orange: #ffd586;
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Montserrat", sans-serif;
+    background-color: #575b74;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+}
+
+.padding__right--clear {
+    padding-right: 0 !important;
+}
+
+.padding__left--clear {
+    padding-left: 0 !important;
+}
+
+.background--blue {
+    background-color: $color-blue;
+}
+
+.background--cyan {
+    background-color: $color-cyan;
+}
+
+.header {
+    background-color: $bg-header;
+    .navbar {
+        padding: 50px 0 40px 0;
+        display: flex;
+        justify-content: space-between;
+        .nav__brand {
+            text-transform: uppercase;
+            flex: 2;
+
+            span:first-child {
+                padding-right: 12px;
+                margin-right: 12px;
+                border-right: 1px solid #717689;
+                font-size: 18px;
+
+                & > a {
```

```suggestion
                > a {
```
Thêm dấu `&` vào làm cái gì vậy